contracts/core/test/octane/Preinstalls.t.sol

```diff
@@ -2,18 +2,55 @@
 pragma solidity =0.8.24;
 
 import { Preinstalls } from "src/octane/Preinstalls.sol";
-import { Test, Vm } from "forge-std/Test.sol";
-
-interface IEIP712 {
-    function DOMAIN_SEPARATOR() external view returns (bytes32);
-}
+import "./PreinstallInterfaces.sol";
+import { AllocPredeploys } from "script/genesis/AllocPredeploys.s.sol";
+import { Test, Vm, console2 } from "forge-std/Test.sol";
```

Comment:
think we can remove console2

we have linting for unused imports but we ignore lints on tests files because was noisy. wish solhint gave us more config options , rather than just ignoring directories entirely. you have any other sol linters you like?

Reply:
Removed console2. Unfortunately I have not used any solidity linters other than `forge fmt`. I know there's solidhunter from Osmium, but its just a faster (not completely featured) solhint.